py_gtfs_rt_ingestion/lib/convert_gtfs_rt.py

```diff
@@ -82,6 +83,11 @@ def convert(self) -> None:
                 # only count table if it contains data
                 if table.num_rows > 0:
                     table_count += 1
+
+                # limit number of tables produced on each event loop
+                if table_count >= max_tables_to_convert:
+                    break
+
```

Comment:
exit, if max tables reached for event loop, do not archive or error remaining files